labui.js.org

ZTL-UwU opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
@ZTL-UwU I've fixed the format of the request for you and added the requested subdomain to JS.ORGs zonefile. But you have to fix a asset/path issue to give me a chance to have a look at your content.
Please keep this PR open.

ZTL-UwU commented:
> @ZTL-UwU I've fixed the format of the request for you and added the requested subdomain to JS.ORGs zonefile. But you have to fix a asset/path issue to give me a chance to have a look at your content. Please keep this PR open.

Thanks! The path problem is now fixed. The project is under development and I'm now putting stuff into it.

ZTL-UwU commented:
It seems like routify requires files starting with `_`, but github pages does not allow that. I will have to rewrite the whole thing with an another router. It might take some more time, I'm sorry.

indus commented:
LGTM
Welcome https://labui.js.org
